lib/std/fs/wasi.zig

```diff
@@ -54,3 +70,23 @@ pub fn preopensAlloc(gpa: Allocator) Allocator.Error!Preopens {
         names.appendAssumeCapacity(name);
     }
 }
+
+// used in Preopens.findFileAndOpen()
+pub fn wasiPathPrefixMatches(prefix: []const u8, path: []const u8) bool {
```

Comment:
this is not intended to be public, but I could not figure out how to run a test from `std/fs/test.zig` without making it public, nor how to run a test from `std/fs/wasi.zig`. I am sure I am just missing something silly

Reply:
You can keep the tests in `std/fs/wasi.zig`. The tests will run as long as they're being referenced. So you could add `comptime { _ = wasi; }` somewhere at the top level in `std/fs/test.zig`

Reply:
TIL